File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/c/cwe-476/code_before3.c
// From cwe-snippets, snippets_90/non-compliant/C/cwe-0476/struct_18.c

void func()
{
    twoIntsStruct * data;
    goto source;
source:
    data = NULL;
    goto sink;
sink:
    printIntLine(data->intOne);
}

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/java/cwe-476/code_before3.java
// From cwe-snippets, snippets_90/non-compliant/Java/cwe-0476/Integer_06.java

public void func() throws Throwable
    {
        Integer data;
        if (PRIVATE_STATIC_FINAL_FIVE==5)
        {
            data = null;
        }
        else
        {
            data = null;
        }

        if (PRIVATE_STATIC_FINAL_FIVE==5)
        {
            IO.writeLine("" + data.toString());
        }
    }